### DISCUSSION TITLE
How would I make an account for a faucet?

### DISCUSSION THREAD

**Discussion by Grantrocks** [revoxhere/duino-coin#1508] [Q&A]
I have made a duino-coin faucet, wallet and more. I have no idea however how i could make an account that is for a faucet. Do i just make another account and verify it like normally? Or is there another way that i have to go and make an account. I would really like to know this answer because i want to get the faucet available as soon as possible for everyone to use.

**Comment by ivanrpz** [comment]
Hi, I'm the owner of https://duino-faucet.com and I use my personal account with the faucet, there aren't any other type of accounts
